its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1098,23 +1182,25 @@ private String getFirstHotspotKey(String projectKey) throws InvalidProtocolBuffe
     }
 
     @Test
+    @OnlyOnSonarQube(from = "9.7")
```

Comment:
What has changed for not supporting older SQ versions?

Reply:
addressed in [4989ca3](https://github.com/SonarSource/sonarlint-core/pull/799/commits/4989ca37baf60cf950245a99149ed3079780e0fa)